blue-xdma/src/include/XDMADescriptorGenerator.bsv

```diff
@@ -0,0 +1,159 @@
+import FIFOF::*;
+import Clocks :: *;
+import GetPut :: *;
+import BlueAXI :: *;
+import BlueLib :: *;
+import AXI4LiteMaster :: *;
+import StmtFSM :: *;
+`include "Config.defines"
+
+typedef struct {
+    Bit#(28) length;
```

Comment:
no hard-coded literal value

Reply:
Since this is an interface exposed by XDMA IP core, the specification is fixed and does not need to be changed into a variable form.